libs/stats/odc/stats/model.py

```diff
@@ -234,37 +236,63 @@ def metadata_path(self, relative_to: str = 'dataset', ext: str = 'yaml') -> str:
     def render_metadata(self, ext: str = EXT_TIFF,
                         processing_dt: Optional[datetime] = None) -> Dict[str, Any]:
         """
-        Put together EO3 metadata document for the output of this task.
+        Put together STAC metadata document for the output of this task.
         """
         if processing_dt is None:
             processing_dt = datetime.utcnow()
 
         product = self.product
         geobox = self.geobox
         region_code = product.region_code(self.tile_index)
+        inputs = list(map(str, self._lineage()))
+
         properties = deepcopy(product.properties)
 
         properties.update(self.time_range.to_stac())
         properties['odc:processing_datetime'] = format_datetime(processing_dt, timespec='seconds')
         properties['odc:region_code'] = region_code
+        properties['odc:lineage'] = dict(inputs=inputs)
+        properties['odc:product'] = product.name
+        properties['proj:epsg'] = geobox.crs.epsg
+        properties['platform'] = "sentinel-2"
```

Comment:
don't like that at all, is landsat WOFS going to be platform `sentinel-2`?

Reply:
Good point.